INSTALL

```diff
@@ -1,8 +1,11 @@
 GstPEAQ - Installation Instructions
 -----------------------------------
+Extra dependency of git version:
+If you clone from git, you need to install the following packages (in debian/ubuntu):
+sudo apt-get install -y git2cl gtk-doc-tools w3-dtd-mathml libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good
```

Comment:
That list is not complete and further, most of it also applies when building from a release tarball. Maybe this should get a (sub-)headline "Prerequisites" or "Requirements" that gives a slightly more detailed list of what is required when.

But I agree this is useful information to add.

Reply:
So I've tried this against the `debian:bookworm-slim` docker image, and I think one needs:
- libgstreamer-plugins-base1.0-dev
- make
- clang or gcc

(and some transitive dependencies of those) for building from the tarball.

For building from git instead of using a release tarball, one additionally needs:
- automake
- git
- git2cl
- libtool

For running (all) tests, one additionally needs:
- gstreamer1.0-plugins-base
- gstreamer1.0-plugins-good
- gstreamer1.0-tools

For building the documentation, one needs:
- gtk-doc-tools
- w3c-sgml-lib (formerly w3-dtd-mathml)

Reply:
Notes are added. thank you.